ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java

```diff
@@ -61,6 +65,20 @@ public void setBorderStyle(ReactViewGroup view, @Nullable String borderStyle) {
     view.setBorderStyle(borderStyle);
   }
 
+  @ReactProp(name = "hitSlop")
+  public void setHitSlop(final ReactViewGroup view, @Nullable ReadableMap hitSlop) {
+    if (hitSlop == null) {
+      view.setTag(R.id.hitSlopTag, null);
+    } else {
+      HashMap<String, Float> hitSlopPixels = new HashMap<>();
+      hitSlopPixels.put("top", PixelUtil.toPixelFromDIP(hitSlop.getDouble("top")));
+      hitSlopPixels.put("bottom", PixelUtil.toPixelFromDIP(hitSlop.getDouble("bottom")));
+      hitSlopPixels.put("left", PixelUtil.toPixelFromDIP(hitSlop.getDouble("left")));
+      hitSlopPixels.put("right", PixelUtil.toPixelFromDIP(hitSlop.getDouble("right")));
+      view.setTag(R.id.hitSlopTag, hitSlopPixels);
```

Comment:
Why not just add an extra property to the `ReactViewGroup` and assing it there? Calling `setTag` this way will often allocate an extra `SparseArray`.